subsys/net/l2/openthread/Kconfig

```
@@ -289,6 +289,13 @@ config OPENTHREAD_TMF_ADDRESS_CACHE_ENTRIES
 	help
 	  The number of EID-to-RLOC cache entries.
 
+config OPENTHREAD_TMF_ADDRESS_CACHE_MAX_SNOOP_ENTRIES
+	int "The maximum number of EID-to-RLOC cache entries"
```

I think it makes sense to add a default value?

```suggestion
	int "The maximum number of EID-to-RLOC cache entries"
	default 2
```

---

fixed.